Removed duplicate Program Files entry in cmds

dakoer opened the pull request:
Commands listed "program files" twice.  Removed second reference to Program Files folder since %programFiles% was specified.
i.e. %programFiles%\Program Files\ (duplicate) to just %programFiles%\

azurecla commented:
Hi __@mingfeiy__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (yanmf). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

dakoer committed:
Removed duplicate Program Files entry in cmds

Commands listed "program files" twice.  Removed second reference to Program Files folder since %programFiles% was specified.
i.e. %programFiles%\Program Files\ (duplicate) to just %programFiles%\
